landing_page/Gemfile

```diff
@@ -50,9 +52,12 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rails-controller-testing'
```

Comment:
Just wondering, do you need to add this gem here as well? I may be wrong, but I think that adding it to the `group :development :test` above means that it's already there when you install develop.

Reply:
Good catch, I'll fix that now

Reply:
Resolved in commit 07d7e2f4dd05baad296d964d81b1d24900b8a7d0